packages/client/src/workflow-client.ts

```diff
@@ -872,9 +866,9 @@ export class WorkflowClient {
           },
           // Safe to convert to number, max history length is 50k, which is much less than Number.MAX_SAFE_INTEGER
           historyLength: raw.workflowExecutionInfo!.historyLength!.toNumber(),
-          startTime: tsToDate(raw.workflowExecutionInfo!.startTime!),
-          executionTime: optionalTsToDate(raw.workflowExecutionInfo!.executionTime),
-          closeTime: optionalTsToDate(raw.workflowExecutionInfo!.closeTime),
+          startTime: tsToDate(raw.workflowExecutionInfo!.startTime!), // eslint-disable-line deprecation/deprecation
+          executionTime: optionalTsToDate(raw.workflowExecutionInfo!.executionTime), // eslint-disable-line deprecation/deprecation
+          closeTime: optionalTsToDate(raw.workflowExecutionInfo!.closeTime), // eslint-disable-line deprecation/deprecation
```

Comment:
What's deprecated here?

Reply:
ah, thanks. the time functions. i'll change it to import from `common/lib/time`